src/testHelpers/jest-setup.js

```diff
@@ -1,5 +1,9 @@
 import fetch from 'jest-fetch-mock';
 import path from 'path';
+import { TextEncoder, TextDecoder } from 'util';
```

Comment:
Don't think this encoder stuff is needed tbh.

Reply:
Nevermind, looks like it is.

Reply:
it was removed from jsdom v16

https://stackoverflow.com/questions/68468203/why-am-i-getting-textencoder-is-not-definedin-jest#answer-68468204